packages/json_decoder/json_decoder.0.1.1/descr

```diff
@@ -0,0 +1,8 @@
+JSON Decoder based on Elm's Json.Decode
+Release %%VERSION%%
```

Comment:
Does this need to be substituted?

Reply:
Probably. New to this but will get it sorted